README.md

```diff
@@ -22,24 +22,25 @@ However, if you want to rely on other roles/management methods to deploy/manage
 
 ## Requirements
 This role requires:
-- Ansible 2.0
+- Ansible 2.3
```

Comment:
I would say this makes it a `### Breaking Changes` in the changelog if this is true. If this is just the tested version they should be called out differently.

Reply:
Any thoughts on how to call it out? I'm not actually breaking 2.0 compatibility more-so, we won't actively be testing for it since it's been long out of the Ansible supported status: http://docs.ansible.com/ansible/latest/release_and_maintenance.html#release-status

Reply:
Then the requirements should stay at `2.0` if you are continuing to support it. I would call out tested versions vs supported versions when they differ. I know it seems like bikeshedding but it is an important distinction.

Reply:
You already have a section talking about testing just add something about we test with ansible version `2.3`

Reply:
If it is outside of the support status and there will be no further support on your end for it then yes bump up required. Otherwise people will submit issues and you will just say sorry we dont support EOL versions. I think that there should be a section talking about version support. For example for the plugins: https://github.com/sensu-plugins/documentation/blob/master/docs/FAQ.md#what-is-the-policy-on-supporting-end-of-lifeeol-ruby-versions

Reply:
Yeah, cool, so I'd like to start getting this role into the position of 'We actively support the last two stable releases of Ansible' (which would be 2.4 and 2.3 compatibility). However, I have no intention of breaking anything in this role with Ansible 2.2 and older. I feel like limiting the burden on the maintainers to the last two stable releases is fair, and also aligns with Ansible's support on their side. I really wouldn't like to see us attempting to get this role to work on 2.4 while keeping it working on 2.0 for the long term. Additionally, I feel uncomfortable stating that the minimum Ansible version is 2.0 when we only actually verify on each PR that 2.3/2.4 work (although this does admittedly need to be fleshed out more). 

Does that seem fair @majormoses? I can add a blurb about version support like you have for `sensu-plugins` as well to make this explicit for folks.

Reply:
:+1: